src/pipecat/processors/filters/stt_mute_filter.py

```diff
@@ -124,6 +139,10 @@ async def _should_mute(self) -> bool:
                         self._first_speech_handled = True
```

Comment:
Should it still remain here, now that we are changing this value inside BotStoppedSpeakingFrame ?

Reply:
Ok, I think I see the differences between them. 👍